src/jshint.js

```diff
@@ -1265,7 +1264,7 @@ var JSHINT = (function() {
             warning("W121", left, nativeObject);
         }
 
-        if (left.identifier) {
+        if (left.identifier && !left.isMetaProperty) {
```

Comment:
I saw your previous comment that you deleted about am I sure - the answer is no, I haven't found the exact place in the spec that says it can't, I got the information from kangax's compatibility table - https://kangax.github.io/compat-table/es6/#new.target_can%27t_be_assigned_to so if you are in doubt too, please check!